src/components/DataDecorator/Decorator/Decorator.js

```diff
@@ -64,13 +64,13 @@ class Decorator extends Component {
     const { path, classes } = getDecoratorProps(score, scoreThresholds, active);
     const decorator = this.renderDecorator(noIcon, path, inline);
     const decoratorClasses = classnames(namespace, classes, className, {
-      [`${namespace}--link`]: this.props.href,
-      [`${namespace}--active`]: this.props.active,
+      [`${namespace}--link`]: this.props.href && !inert,
+      [`${namespace}--active`]: this.props.active && !inert,
       [`${namespace}--inert`]: this.props.inert,
       [`${namespace}--inline`]: this.props.inline,
     });
 
-    if (this.props.href) {
+    if (this.props.href && !inert) {
```

Comment:
@diego-codes When you get a chance, I was curious if you could take a look at the above & see if this change would be all right with you? I feel like this makes sense since `inert` is meant to mean that a component is not interactive. So it shouldn't get interactive classes (like `--link` or `--active`) and it also shouldn't render as an anchor.

Currently, an `inert` decorator could still render as an `<a>` if an `href` is provided 😬

Reply:
Yeah, that makes sense to me. `inert` would remove any interactivity, which makes sense. I think a potential API update is to think of props being additive than subtractive, which `inert` is IMO. 

Perhaps a decorator is inert when no `onClick` or `href` is provided or something like that? Inertion (is that a word?) could be implied based on other props. That way, you don't end up with different props as we have now where `inert` and `href` could contradict each other.

Reply:
hey @diego-codes, just to clarify, you are suggesting to update and remove the `inert` prop and make this component's interactivity based on `href || onClick` ?

Reply:
Yep! If the consumer does not provide a prop to make the decorator interactive, then it should be implied that it is inert.